utilities/bash/dot_profile

```diff
@@ -1277,6 +1285,87 @@ sixDeskSetBOINCVars(){
     # for megaZip
     [ -n "${megaZipPath}" ] || export megaZipPath=$boincdir/boinc/upload
 }
+
```

Comment:
If I correctly understand, the difference between this function and the original one for BOINC is the addition of the last two lines, ie where you define remoteHost and htboincport, right? If this is the case, then I would stay with the other function, and add a conditional statement on the platform. If you do so, please remember to remove the call to sixDeskSetHTBOINCVars in run_six.sh

Moreover, I would let the function define the two variables and assign their values only if they are not defined.